src/app/view/components/login/dot-login-component/dot-login.module.ts

```diff
@@ -1,16 +1,17 @@
 import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
 import { RouterModule, Routes } from '@angular/router';
-import { FormsModule } from '@angular/forms';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
 import { DotLoadingIndicatorModule } from '../../_common/iframe/dot-loading-indicator/dot-loading-indicator.module';
 import { ButtonModule, CheckboxModule, DropdownModule, InputTextModule } from 'primeng/primeng';
 
 import { LoginContainerComponent } from './login-container.component';
-import { LoginComponent } from './login.component';
+import { DotLoginComponent } from '@components/login/dot-login-component/dot-login.component';
 
 import { SharedModule } from '@shared/shared.module';
 import { DotDirectivesModule } from '@shared/dot-directives.module';
 import { MdInputTextModule } from '@directives/md-inputtext/md-input-text.module';
+import { DotFieldValidationMessageModule } from '@components/_common/dot-field-validation-message/dot-file-validation-message.module';
 
 const routes: Routes = [
```

Comment:
All [this routes](https://github.com/dotCMS/core-web/blob/issue-16167-redesing-login-page/src/app/app-routing.module.ts#L15-L28) should live in the login module, right?

Reply:
moved